install/kubernetes/cilium/templates/hubble-ui/deployment.yaml

```diff
@@ -46,7 +46,7 @@ spec:
         imagePullPolicy: {{ .Values.hubble.ui.frontend.image.pullPolicy }}
         ports:
         - name: http
-          containerPort: 8080
+          containerPort: 8081
```

Comment:
any reason we change the container port here ?

Reply:
Yes, there is the reason. Previously nginx config for UI was bundled inside docker image, and [this used 8080 port](https://github.com/cilium/hubble-ui/blob/0e7710af20bc64a893ed8ccbc9f9463451b418d0/server/nginx-hubble-ui-frontend.conf#L2), envoy transferred http traffic to 8080 frontend port as well, that's why we used 8080 here in deployment. 

We also have k8s service for UI which targets to 8081 port (this was an exposed envoy port). For more compatibility I decided to not to change service port and now nginx config in configmap listens on 8081 port as well. I don't think that this way we break something, because it wasn't possible to target directly to frontend nginx container and have UI working, because anyway you need to proxy grpc-web traffic at first place.